CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - inconsistent messaging in SDCcc logs ("No problems were found" and "Test run was invalid" one after another.)
 - incorrect behavior of the configuration option SDCcc.SummarizeMessageEncodingErrors
 - SequenceIds are now ordered by the timestamp of the first message that used them
+- test for BICEPS.R5003 does not check MdDescription and MdStateVersion anymore
```

Comment:
I feel like this needs to be argued better, probably in the PR description and not in the changelog. Why is this acceptable? The PR description says because the versions aren't transmitted via reports, but why does this mean the test doesn't have to check it?

Reply:
Changed PR description and link to it here